Modules/termios.c

```diff
@@ -82,28 +80,29 @@ static PyObject *
 termios_tcgetattr(PyObject *self, PyObject *args)
```

Comment:
Would you mind to rename self parameter to module? Same remark for other functions that you modified to add a PyModule_GetState() call.

Reply:
@vstinner sure, the CI is stuck anyways so I need to push something anyways